tutoraspects/templates/aspects/jobs/init/lms/init-lms.sh

```diff
@@ -6,15 +6,13 @@
     --client-id {{ SUPERSET_OAUTH2_CLIENT_ID }} \
     --client-secret {{ SUPERSET_OAUTH2_CLIENT_SECRET }} \
     --scopes "user_id" \
-    --update \
```

Comment:
With this in place the plugin will overwrite any changes operators make to their configuration on any upgrade. I think it's better to put the difficulty on developers than have that confusing behavior in production.